scripts/core-suite-deployments/1_reference_mainnet_flagship_V3ExplorationsCore_deployer.ts

```diff
@@ -0,0 +1,288 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import { ethers } from "hardhat";
+// flagship
+import { GenArt721CoreV3__factory } from "../contracts/factories/GenArt721CoreV3Explorations__factory";
+import { AdminACLV0__factory } from "../contracts/factories/AdminACLV0__factory";
+import { BasicRandomizerV2__factory } from "../contracts/factories/BasicRandomizerV2__factory";
+// minter suite
+import { MinterFilterV1__factory } from "../contracts/factories/MinterFilterV1__factory";
+import { MinterSetPriceV2__factory } from "../contracts/factories/MinterSetPriceV2__factory";
+import { MinterSetPriceERC20V2__factory } from "../contracts/factories/MinterSetPriceERC20V2__factory";
+import { MinterDALinV2__factory } from "../contracts/factories/MinterDALinV2__factory";
+import { MinterDAExpV2__factory } from "../contracts/factories/MinterDAExpV2__factory";
+import { MinterMerkleV1__factory } from "../contracts/factories/MinterMerkleV1__factory";
+import { MinterHolderV1__factory } from "../contracts/factories/MinterHolderV1__factory";
+
+// delay to avoid issues with reorgs and tx failures
+import { delay } from "../util/utils";
+const EXTRA_DELAY_BETWEEN_TX = 5000; // ms
+
+/**
+ * This script was created to deploy the V3 core Explorations contracts,
+ * including the associated minter suite, to the Ethereum mainnet.
+ * It is intended to document the deployment process and provide a
+ * reference for the steps required to deploy the V3 core contract suite.
+ * IMPORTANT: This deploys a basic randomizer, which may be changed after
+ * deployment by the configured superAdmin.
+ */
+//////////////////////////////////////////////////////////////////////////////
+// CONFIG BEGINS HERE
+//////////////////////////////////////////////////////////////////////////////
+const tokenName = "Art Blocks Explorations";
+const tokenTicker = "BLOCKS_EXPLORATIONS";
```

Comment:
Do you think we need a shorter ticker here?

Reply:
"EXPLORE" may be better–let me sanity check with a couple folks on leadership.

Reply:
lmk and I will update this to whatever is best 👍

Reply:
as discussed in DM, updated token symbol to "EXPLORE", kept "Art Blocks Explorations" 🎉